data/catalysts.ts

```diff
@@ -0,0 +1,39 @@
+import {BigNumber} from '@ethersproject/bignumber';
+function sandWei(amount: number) {
+  return BigNumber.from(amount).mul('1000000000000000000').toString();
+}
+
+export = [
+  {
+    name: 'Common',
+    symbol: 'COMMON',
+    sandMintingFee: sandWei(1),
```

Comment:
We should remove 
- sandMintingFee
- sandUpdateFee
- quantityRange

and leave only name, symbol and maxGems